pkgs/top-level/qt5-packages.nix

```
@@ -210,7 +210,9 @@ in (kdeFrameworks // plasmaMobileGear // plasma5 // plasma5.thirdParty // kdeGea
 
   quazip = callPackage ../development/libraries/quazip { };
 
-  qwt = callPackage ../development/libraries/qwt/6.nix { };
+  qwt = callPackage ../development/libraries/qwt/default.nix { };
+
+  qwt6_1 = callPackage ../development/libraries/qwt/6_1.nix { };
```

```suggestion
  qwt_6 = callPackage ../development/libraries/qwt/6.nix { };
```
Or do we want to rename qwt to qwt_5?

---

That’s a good question. It seems like nothing actually needs qwt5 anymore, so I figured qwt would be 6.2+ going forward, the new qwt6_1 would be for derivations that have to stick to the older version, and 5 could get dropped entirely.

---

All other issues fully resolved. With my latest force-push this still has `qwt` as the newest version (6.2), with `qwt6_1` as the primary fallback for software that isn't yet working with `qwt 6.2.0`. It appears that `elmerfem` is the only other software still relying on `qwt6_qt4` which remains in `all-packages.nix`.

Let me know if you'd still like to retain `qwt5` inside `qt5-packages`. The last 5.x.x release was 5.2.3 in November of 2012, so it might be okay to drop.

---

If it is not used anywhere we can safely remove it.